server/graphql/types/src/types/item.rs

```diff
@@ -56,6 +56,18 @@ impl ItemNode {
         VenCategoryType::from_domain(&self.row().ven_category)
     }
 
+    pub async fn is_vaccine(&self) -> &bool {
+        &self.row().is_vaccine
+    }
+
+    pub async fn default_pack_size(&self) -> f64 {
+        self.row().default_pack_size
+    }
+
+    pub async fn doses(&self) -> i32 {
+        self.row().doses
+    }
```

Comment:
These fields are now stored on the om item table, rather than parsing from the legacy record